components/Days.vue

```diff
@@ -0,0 +1,192 @@
+<template>
+  <div class="week-box">
+    <div class="heading">
+      <img src="~/assets/ankur.png" alt="userpic" />
+      <h1>Your goals</h1>
+    </div>
+    <h2 class="week-heading">This week</h2>
+    <div class="days">
+      <div class="day-box">
+        <div class="day">Mon</div>
+        <button class="complete"></button>
+      </div>
+      <div class="day-box">
+        <div class="day">Tue</div>
+        <button class="complete"></button>
+      </div>
+      <div class="day-box">
+        <div class="day">Wed</div>
+        <button class="complete"></button>
+      </div>
+      <div class="day-box">
+        <div class="day">Thu</div>
+        <button class="complete"></button>
+      </div>
+      <div class="day-box">
+        <div class="day">Fri</div>
+        <button class="inprogress"></button>
+      </div>
+      <div class="day-box">
+        <div class="day">Sat</div>
+        <button class="incomplete"></button>
+      </div>
+      <div class="day-box">
+        <div class="day">Sun</div>
+        <button class="incomplete"></button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Days',
+}
+</script>
+
+<style scoped>
+@import url('https://fonts.googleapis.com/css2?family=Saira+Semi+Condensed:wght@800&display=swap');
```

Comment:
Same font as been imported here as well. It was also imported in `Clock.vue`

Reply:
done.